modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -241,10 +242,16 @@ public String getContentDashboardItemSubtypeItemSelectorURL() {
 						Class<?> genericClass = GenericUtil.getGenericClass(
 							contentDashboardItemSubtype);
 
+						ClassNameClassPKInfoItemIdentifier
+							classNameClassPKInfoItemIdentifier =
+								(ClassNameClassPKInfoItemIdentifier)
+									infoItemReference.getInfoItemIdentifier();
```

Comment:
@cgoncas, we went this way because we saw that it was always a  ClassNameClassPKInfoItemIdentifier. If not, we can always check for the instance and use infoItemreference.getClassPK as the default value

Reply:
Hey @jkappler !!

We do exactly this in other parts of the Content Dashboard, so for me it is super OK.

Thanks!!!